kie-pmml/src/main/java/org/kie/pmml/pmml_4_2/model/AbstractModel.java

```
@@ -430,6 +434,90 @@ public PMML4Unit getOwner() {
+    	            results.put(key, new String(baos.toByteArray()));
+    	        } catch (TemplateError te) {
+    	            te.printStackTrace();
+    	        } catch (TemplateRuntimeError tre) {
+    	            // need to figure out logging here
+    	            tre.printStackTrace();
+    	        }
+    			
+    		});
+    	}
+    	Map<String,OutputField> flds = this.getOutputFieldMap();
+    	if (flds != null && !flds.isEmpty()) {
+    		flds.values().forEach(of -> {
+    			Map<String,Object> vars = new HashMap<>();
+    			String className = helper.compactUpperCase(of.getName());
+    			vars.put("name", of.getName());
+    			vars.put("className", className);
+    			vars.put("displayValue", className);
+    			vars.put("packageName", packageName);
+    			vars.put("modelId", this.getModelId());
+    			String typeName = "String";
+    			if (of.getDataType() != null) {
+    				typeName = helper.mapDatatype(of.getDataType(), true);
+    			} else {
+    				if (RESULTFEATURE.PROBABILITY.equals(of.getFeature())) {
+    						typeName = helper.mapDatatype(DATATYPE.DOUBLE, true);
+    				} else if (of.getTargetField() != null) {
+    					PMMLDataField ddf = getOwner().findDataDictionaryEntry(of.getTargetField());
+    					if (ddf != null) {
+    						typeName = ddf.getType();
+    					}
+    				}
+    			}
+    			vars.put("typeName", typeName);
+    	        ByteArrayOutputStream baos = new ByteArrayOutputStream();
+    	        try {
+    	            TemplateRuntime.execute(templateRegistry.getNamedTemplate(templateId),
+    	                                    null,
+    	                                    new MapVariableResolverFactory(vars),
+    	                                    baos);
+    	            String key = packageName+"."+className;
+    	            results.put(key, new String(baos.toByteArray()));
+    	        } catch (TemplateError te) {
+    	            te.printStackTrace();
```

Same as previous comments about swallowing exceptions.